lib/data_hygiene/bulk_organisation_updater.rb

```diff
@@ -34,11 +34,22 @@ def process_row(row)
     end
 
     def find_document(row)
-      slug = row.fetch("Slug")
+      slug = row.fetch("Slug")&.strip
+      document_type = row.fetch("Document type")&.strip
 
       return if slug.blank?
 
-      document = Document.find_by(slug: slug)
+      if document_type.blank?
+        documents = Document.where(slug: slug.strip).to_a
+
+        if documents.length > 1
+          puts "error: ambiguous slug: #{slug} (document_types: #{documents.map(&:document_type).uniq})"
```

Comment:
I'd leave off the `.uniq` - if we end up in a situation where a slug is applied to two documents of the same type, this error will highlight the issue, whereas with `uniq` it would not.

Reply:
That should be impossible https://github.com/alphagov/whitehall/blob/master/db/schema.rb#L290 and generally I trust database constraints.

Reply:
Which I guess means the `.uniq` is redundant, so I'll remove it anyway.